src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function isNotNull<T>(arg: T): arg is NonNullable<T> {
+  return arg !== null
+}
```

Comment:
TypeScript should no longer need this. If it does then override the type-checker instead of creating a new file for a one-liner.